### ISSUE TITLE
Feat: Workflow for PR preview

### ISSUE THREAD

**Issue by pooranjoyb** [gdscsmit/gdscsmit.github.io#76]
### What feature?

Create a workflow to preview a PR live in some hosting platform. (Github-Pages, Vercel, Netlify, etc)

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I want to work on this issue

**Comment by pooranjoyb** [comment]
@sid86-dev Wanna work on this?

**Comment by sid86-dev** [comment]
Absolutely, you may assign me to this issue, I'll definitely work on this and keep you updated on the progress.

**Comment by pooranjoyb** [comment]
Sure 🚀 
Btw, why did you linked separate issues and pushed changes to different PR tho? 😆 
@sid86-dev

**Comment by sid86-dev** [comment]
Currently, I'm actively engaged in multiple projects, and as a result, I encountered some confusion between different issue ids 😅 , especially since it was conflicting with my forked branch.
